db/migrate/20221203190338_create_likes.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Again, considering the given ERD, the reference name should be `author` instead of `user`, hence creating a migration query should be similar to:

```ruby
...
  def change
    add_reference :likes, :author, references: :users, null: false, foreign_key: {to_table: :users}
  end
...